src/test/regress/sql/multi_colocation_utils.sql

```diff
@@ -57,11 +57,6 @@ CREATE FUNCTION shards_colocated(bigint, bigint)
     AS 'citus'
     LANGUAGE C STRICT;
 
-CREATE FUNCTION get_colocated_table_array(regclass)
```

Comment:
well, we should have some tests for this important feature. We could add it either on colocation utils, or, better on `multi_repair_shards`